EncryptionTests/Encryption Tests/CDTQIndexManagerEncryptionTests.m

```diff
@@ -209,4 +210,21 @@ - (void)testCreateQueryIndexManagerWithFixedKeyProviderFailsIfDBIsEncryptedButKe
     XCTAssertNotNil(err, @"There is an error");
 }
 
+- (void)testTextSearchIsAvailable
+{
+    CDTHelperFixedKeyProvider *provider = [CDTHelperFixedKeyProvider provider];
+    CDTDatastore *datastore = [self.factory datastoreNamed:@"is_fts_activated"
+                                 withEncryptionKeyProvider:provider
+                                                     error:nil];
+    
+    CDTQIndexManager *im = [[CDTQIndexManager alloc] initUsingDatastore:datastore error:nil];
+
+    NSSet *compileOptions = [DBQueryUtils compileOptions:im.database];
+    NSSet *ftsCompileOptions = [NSSet setWithArray:@[ @"ENABLE_FTS3", @"ENABLE_FTS3_PARENTHESIS" ]];
+
+    XCTAssertTrue([ftsCompileOptions isSubsetOfSet:compileOptions]);
+
+    XCTAssertTrue([im isTextSearchEnabled], @"It should be activated");
```

Comment:
My initial thought was that perhaps there could be a real test showing FTS working with a document rather than just checking for the compile options and if -isTextSearchEnabled return true

however... @mikerhodes replied that -isTextSearchEnabled should be sufficient.

And I guess we can assume that FMDB gets it right: https://github.com/cloudant/CDTDatastore/blob/master/Classes/common/query/CDTQIndexManager.m#L354-L372.

So, effectively this is a "no comment", but I put it here since you asked me nicely (on Slack).